auth/authn.go

```diff
@@ -1,39 +1,37 @@
 package auth
 
 import (
-	"crypto/x509"
+	"encoding/base32"
 	"encoding/base64"
-	"encoding/pem"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"strconv"
 	"strings"
-	"time"
 
-	"github.com/auth0-community/go-auth0"
 	"github.com/banzaicloud/banzai-types/database"
 	"github.com/banzaicloud/pipeline/cloud"
 	jwt "github.com/dgrijalva/jwt-go"
+	jwtRequest "github.com/dgrijalva/jwt-go/request"
 	"github.com/gin-gonic/gin"
 	"github.com/qor/auth"
 	"github.com/qor/auth/authority"
+	"github.com/qor/auth/claims"
 	"github.com/qor/auth/providers/github"
 	"github.com/qor/redirect_back"
 	"github.com/qor/session/manager"
 	"github.com/satori/go.uuid"
 	"github.com/spf13/viper"
-	"gopkg.in/square/go-jose.v2"
 
 	banzaiConstants "github.com/banzaicloud/banzai-types/constants"
 	banzaiUtils "github.com/banzaicloud/banzai-types/utils"
 )
 
-const jwksUri = "https://banzaicloud.auth0.com/.well-known/jwks.json"
-const auth0ApiIssuer = "https://banzaicloud.auth0.com/"
+const ApiIssuer = "https://banzaicloud.com/"
+const DroneSessionCookie = "user_sess"
+const DroneSessionCookieType = "sess"
+const DroneUserCookieType = "user"
```

Comment:
Please document what other values beside `user` are valid for `DroneUserCookieType`